nanopub/setup_nanopub_profile.py

```diff
@@ -39,12 +39,11 @@ def validate_orcid_id(ctx, orcid_id: str):
                     '(by default HOMEDIR/.nanopub/). '
                     'The profile will also be published to the nanopub servers.')
 @click.option('--keypair', nargs=2, type=Path,
-              prompt=f'If the public and private key you would like to use are not '
-                     f'in {USER_CONFIG_DIR}, provide them here. '
-                     f'If they are in this directory or you wish to generate new keys, '
-                     f'leave empty.',
               help='Your RSA public and private keys with which your nanopubs will be signed',
               default=None)
+@click.option('--newkeys', type=bool, is_flag=True, default=False,
```

Comment:
I added this so we can just call `setup_nanopub_profile` fully non-interactively from the command line (which is much nicer for github action).

Reply:
Yes, this is a much needed option.